recipes/assimp/5.0.x/conanfile.py

```diff
@@ -112,14 +111,61 @@ def configure(self):
         if self.options.shared:
             del self.options.fPIC
 
+    @property
+    def _depends_on_kuba_zip(self):
+        return self.options.with_3mf_exporter
+
+    @property
+    def _depends_on_poly2tri(self):
+        return self.options.with_blend or self.options.with_ifc
+
+    @property
+    def _depends_on_rapidjson(self):
+        return self.options.with_gltf or self.options.with_gltf_exporter
+
+    @property
+    def _depends_on_zlib(self):
+        return self.options.with_assbin or self.options.with_assbin_exporter or \
+               self.options.with_assxml_exporter or self.options.with_blend or self.options.with_fbx or \
+               self.options.with_q3bsp or self.options.with_x or self.options.with_xgl
```

Comment:
none of these `with_foo` options seem to be declared above, and yet the build passed. I am obviously missing something, what is it?

Reply:
They are added a few lines below from a dict called `_format_option_map`, around lines 90-92.